samples/document-snippets/app-profile.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const snippets = {
+  create: (instanceId, appProfileId) => {
+    // [START bigtable_create_app_profile]
+    const Bigtable = require('@google-cloud/bigtable');
+    const bigtable = new Bigtable();
+    const instance = bigtable.instance(instanceId);
+    const appProfile = instance.appProfile(appProfileId);
+
+    appProfile
+      .create()
```

Comment:
Will this work without setting the appProfile type?

Reply:
I feel it should work. 
I saw the constructor for appProfile, 'type' is not required

Reply:
I feel like it won't work. `type` is not required for `getMetadata()`, but `create()` shouldn't work  without it.